dataverse-webapp/src/main/webapp/resources/css/structure.css

```diff
@@ -537,6 +537,53 @@ table[id$='resultsTable'] td > div > div.file-tags-block {
 /* -------- TABLES -------- */
 div.headerless-table .ui-datatable-tablewrapper table > thead {display:none;}
 
+/* -------- Quill TextEditor -------- */
+.ql-editor p {
+    margin: 0 0 10px;
+    padding: 0;
+}
+
+.ql-snow .ql-editor blockquote {
+    padding: 10px 20px;
+    margin: 0 0 20px;
+    font-size: 17.5px;
+    border-left: 5px solid #eee;
+}
+
+.ql-editor sup {
+    position: relative;
+    font-size: 75%;
+    line-height: 0;
+    vertical-align: baseline;
+}
+
+.ql-editor sub, sup {
```

Comment:
Same with this selector. We must do `.ql-editor sub, .ql-editor sup` and remove section for `.ql-editor sup` to avoid duplications.